_source/_data/authors.yml

```
@@ -719,3 +719,10 @@ deepu-sasidharan:
   bio:
     Deepu is a polyglot developer and OSS aficionado. He mainly works with Java, JS, Rust, and Golang. He co-leads JHipster and created the JDL Studio and KDash.
     He's a Developer Advocate for DevOps at Okta. He is also an international speaker and published author.
+
+alisa-duncan: 
+  full_name: Alisa Duncan
+  display_name: Alisa Duncan
+  avatar: avatar-alisa_duncan.jpeg
+  github: https://github.com/alisaduncan
+  twitter: https://twitter.com/alisaduncan
```

```suggestion
  twitter: https://twitter.com/alisaduncan
  
```
I think this needs a newline, but I'm not 100% certain